drizzlepac/wcs_functions.py

```diff
@@ -2,7 +2,7 @@
 
 :Authors: Warren Hack
 
-:License: :doc:`../LICENSE`
+
```

Comment:
Is there a reason to remove the license info? Some institutions recommend adding it to all source files so license information can be preserved and tracked in the future. I've no strong feelings about this. Will look to find some more information.

Reply:
License links have been re-introduced.